ERCS/erc-TBD-permit.md

```diff
@@ -0,0 +1,137 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 7597
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.